marklogic-data-hub-central/ui/src/config/bench.config.ts

```diff
@@ -347,6 +347,25 @@ const loadsXML = {"data" :
   "status" :200
 };
 
+const setupMockAPIs = (axiosMock) => {
+  return axiosMock.get['mockImplementation']((url) => {
+    switch (url) {
+      case '/api/flows':
+        return Promise.resolve(flows)
+      case '/api/models/primaryEntityTypes':
+        return Promise.resolve(primaryEntityTypes)
+      case '/api/steps/ingestion':
+        return Promise.resolve(loadsXML)
```

Comment:
In the UI work, we typically use JSON as the default case. Can we have setupMockAPIs use the loads object (the JSON version) for /api/steps/ingestion instead of loadsXML? That will also make the test data more consistent since the flows test data has a step with a sourceFormat of 'json'.

Reply:
updated to use the loads object.